client/test/router_store.spec.js

```diff
@@ -63,18 +70,17 @@ describe('RouterStore', () => {
   });
 
   beforeEach(() => {
-    Dispatcher.dispatch({
-      type: ActionTypes.ROUTES_INITIALIZE,
-      value: router,
-    });
+    Dispatcher.dispatch({type: ActionTypes.RESET_FOR_TESTS});
```

Comment:
@aenario : Why not keep this event name? What the issue about this? I think it's important not to create specific workflow for test but keep real one to make test as relevant as possible.